app/src/main/java/io/appium/uiautomator2/handler/GetElementAttribute.java

```diff
@@ -164,6 +164,11 @@ private static int getScrollableOffsetByItemCount(AndroidElement uiScrollable, i
             }
             Logger.debug("Determined there were " + itemsPerRow + " items per row");
 
+            if (itemsPerRow == 0) {
+                // Need to exit. Other case we will get an ArithmeticException
+                return 0;
```

Comment:
I'd assume the resulting number of rows will be `itemCount` in such case rather than zero

Reply:
This method should return offset value for provided number of rows. 
itemCount as a result would be wrong. Return zero as a result can avoid to failing the test